src/cosmosdb-preview/azext_cosmosdb_preview/commands.py

```diff
@@ -0,0 +1,81 @@
+
+    cosmosdb_restorable_sql_resources_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableSqlResourcesOperations.{}',
+        client_factory=cf_restorable_sql_resources)
+
+    cosmosdb_restorable_mongodb_databases_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableMongodbDatabasesOperations.{}',
+        client_factory=cf_restorable_mongodb_databases)
+
+    cosmosdb_restorable_mongodb_collections_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableMongodbCollectionsOperations.{}',
+        client_factory=cf_restorable_mongodb_collections)
+
+    cosmosdb_restorable_mongodb_resources_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableMongodbResourcesOperations.{}',
+        client_factory=cf_restorable_mongodb_resources)
+
+    with self.command_group('cosmosdb restorable-database-account', cosmosdb_restorable_database_accounts_sdk, client_factory=cf_restorable_database_accounts, is_preview=True) as g:
+        g.show_command('show', 'get_by_location')
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb', cosmosdb_sdk, client_factory=cf_db_accounts) as g:
+        g.show_command('show', 'get')
+        g.custom_command('restore', 'cli_cosmosdb_restore', is_preview=True)
+        g.custom_command('create', 'cli_cosmosdb_create')
+        g.custom_command('update', 'cli_cosmosdb_update')
+        g.custom_command('list', 'cli_cosmosdb_list')
+
+    with self.command_group('cosmosdb sql restorable-database', cosmosdb_restorable_sql_databases_sdk, client_factory=cf_restorable_sql_databases, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb sql restorable-container', cosmosdb_restorable_sql_containers_sdk, client_factory=cf_restorable_sql_containers, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb sql restorable-resource', cosmosdb_restorable_sql_resources_sdk, client_factory=cf_restorable_sql_resources, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-database', cosmosdb_restorable_mongodb_databases_sdk, client_factory=cf_restorable_mongodb_databases, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-collection', cosmosdb_restorable_mongodb_collections_sdk, client_factory=cf_restorable_mongodb_collections, is_preview=True) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-resource', cosmosdb_restorable_mongodb_resources_sdk, client_factory=cf_restorable_mongodb_resources, is_preview=True) as g:
+        g.command('list', 'list')
```

Comment:
If only one command in these command group, what about use `list-restorable-db`, `list-restorable-resource`...?

Reply:
But you could keep current design if you have concern that there would be more commands to add in these command group

Reply:
sure. Let me change that and send a new iteration.

Reply:
Yes. We might have more commands under each. 
Also, each of the commands have different client factory.
Can commands in the same group use different sdks?

Reply:
yes, you could use it with `command_type` kwargs in command.